### ISSUE TITLE
Automated accessibility check not catching accessibility errors 

### ISSUE THREAD

**Issue by Moro-Code** [sara-sabr/ITStrategy#847]
Automated check reporting 0 violations for even if there is violations.

**Comment by jaysonmc** [comment]
"I will do it by the end of this week"
  - @Moro-Code, 2020-08-19

**Comment by jaysonmc** [comment]
@Moro-Code move to pending and book meeting with the others for after the 31st?

**Comment by Moro-Code** [comment]
Meet on the 31st, could not solve

**Comment by smellems** [comment]
axe-cli test removed from npm test in #1207
